CIP-0050/README.md

```diff
@@ -1,431 +1,239 @@
 ---
-CIP: 50
-Title: Pledge Leverage-Based Staking Rewards
-Status: Proposed
-Category: Ledger
-Authors:
+CIP: 50 
+Title: Pledge Leverage-Based Staking Rewards 
+Category: Ledger / Protocol 
```

Comment:
```suggestion
Category: Ledger
```
1. there isn't a `Protocol` category;
2. there's a `Consensus` category but all the RSS proposals have long been considered as `Ledger` regardless of what effect they might have on consensus;
3. we're constrained to select a single category: because of tooling around CIPs and some practical & editorial issues highlighted here:
- https://github.com/cardano-foundation/CIPs/issues/433